pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -21,6 +21,30 @@ spec:
       {{ if .ProvisionerPriorityClassName }}
       priorityClassName: {{ .ProvisionerPriorityClassName }}
       {{ end }}
+      initContainers:
+        - name: create-logrotate-config
+          image: {{ .CSIPluginImage }}
+          command: ["/bin/sh", "-c"]
+          args:
+            - |
+              echo "Starting init container";
```

Comment:
It occurs to me that the sidecar itself should be able to create this config. That would simplify the setup, make it easier to modify for future devs, and make things less fragile long-term.

Reply:
Updated